crates/nao/src/lib.rs

```diff
@@ -11,14 +11,40 @@ use color_eyre::{
 use tokio::process::Command;
 
 pub struct Nao {
-    host: Ipv4Addr,
+    pub host: Ipv4Addr,
 }
 
 impl Nao {
     pub fn new(host: Ipv4Addr) -> Self {
         Self { host }
     }
 
+    pub async fn has_stable_os_version(&self) -> bool {
+        let os_stable = "5.2.0";
+        let output = self
+            .ssh_to_nao()
+            .arg("cat /etc/os-release")
+            .output()
+            .await
+            .unwrap();
+
+        let stdout = String::from_utf8(output.stdout).unwrap();
```

Comment:
You are creating a `String` but then take a `&str` of it. Iirc you can just use `str::from_utf8()`.

In addition, we decided in a recent Dev-Meeting that we aim to use all things instead of using qualified paths. Insert `use str::from_utf8;` to your imports and remove the `str::` prefix from your call.